### ISSUE TITLE
Odd error string in chai-vague-errors

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [microsoft/tslint-microsoft-contrib#660]
### Bug Report

-   **`tslint-microsoft-contrib` version**: 6.0.0
-   **TSLint version**: 5.11.0
-   **TypeScript version**: 3.1.2
-   **Running TSLint via**: CLI

#### TypeScript code being linted

```tsx
expect(something === undefined).to.equal(false, 'something should not have been set');
expect(something !== undefined).to.equal(true, 'something should not have been set');
```

with `tslint.json` configuration:

```json
{
    "extends": "tslint-microsoft-contrib",
    // ...
    "rules": {
        "chai-vague-errors": true
    }
}
```

#### Actual behavior

```
Found chai call with vague failure message. Move the strict equality comparison from the expect call into the assertion value
Found chai call with vague failure message. Move the strict inequality comparison from the expect call into the assertion value
```

#### Expected behavior

```
Found chai call with vague failure message. Move the strict equality comparison from the expect call into the assertion value
Found chai call with vague failure message. Move the strict inequality comparison from the expect call into the assertion value. 
```

Note the period and space in the second error message. Seems odd.

**Comment by JoshuaKGoldberg** [comment]
I'll take this on as a side mission to #489.

**Comment by JoshuaKGoldberg** [comment]
☠️  *It's time!* ☠️ 

Per #876, this repository is no longer accepting feature pull requests. TSLint is being deprecated and we recommend you switch to https://typescript-eslint.io.

Thanks for open sourcing with us, everyone!
